feat:(n-carousel): add dotPosition prop (#366)
TuSimple/naive-ui: src/carousel/demos/enUS/dotPosition.demo.md

> # 更改指示点位置

Reviewer: ```suggestion
# Set Dot Placement
```

> Set 'dot-placement' to change the indication point position.

Reviewer: ```suggestion
Set `dot-placement` to change the indication point position.
```

> # Set `dot-placement` to change the indication point position.

Reviewer: ```suggestion
Set `dot-placement` to change the indication point position.
```

```html

> <n-carousel dot-placement="left" style="height:240px" autoplay>

Reviewer: ```suggestion
<n-carousel dot-placement="left" style="height: 240px;" autoplay>
```
